src/commands/seek.rs

```diff
@@ -53,7 +53,7 @@ pub async fn seek(
     create_response(
         &ctx.http,
         interaction,
-        &format!("Seeked current track to **{}**!", seek_time),
+        &format!("⏭  Seeked current track to **{}**!", seek_time),
```

Comment:
⏩ (meaning fast-forward) makes more sense here as opposed to ⏭️ (meaning [next track](https://emojipedia.org/emoji/%E2%8F%AD%EF%B8%8F/)).